go.mod

```diff
@@ -27,7 +27,7 @@ require (
 require (
 	cosmossdk.io/errors v1.0.0-beta.7
 	cosmossdk.io/math v1.0.0-beta.3
-	github.com/celestiaorg/rsmt2d v0.7.0
+	github.com/celestiaorg/rsmt2d v0.8.0
```

Comment:
[question][optional] does this PR depend on anything from rsmt2d v0.8.0? I don't think it does so this dependency upgrade could be extracted to a stand-alone PR to minimize the diff and isolate independent changes.

Reply:
Must have bumped when I ran go mod tidy. No it doesn't depend on rsmt2d. I can extract it out

Reply:
Ah it's in `main`. I just haven't updated

Reply:
Resolved by https://github.com/celestiaorg/celestia-app/pull/1219/commits/eff692744563c57e659ce8d08ac91f80056ce0a6